packages/w3/index.js

```diff
@@ -92,6 +103,55 @@ export async function get (cid, opts) {
   await writeFiles(res.unixFsIterator(), opts.output)
 }
 
+/**
+ * Print out all the uploads in your account by data created
+ *
+ * @param {object} opts
```

Comment:
```suggestion
 * @param {object} [opts]
```

Let's also make opts optional?